backend/layers/business/business.py

```diff
@@ -5,12 +5,14 @@
 from datetime import datetime
 from functools import reduce
 from typing import Dict, Iterable, List, Optional, Set, Tuple
+from backend.common.corpora_config import CorporaConfig
```

Comment:
This import should be moved to the method definition. Importing `CorporaConfig` has an implicit dependency on boto3 which we should avoid. I am not sure this is still enforced throughout the codebase but we should try to keep the business layer dependency free.